oktoberfest/runner.py

```diff
@@ -59,11 +61,20 @@ def _preprocess(spectra_files: List[Path], config: Config) -> List[Path]:
             msms_output.mkdir(exist_ok=True)
             internal_search_file = msms_output / "msms.prosit"
             tmt_label = config.tag
+            custom_var_mods = config.var_mods
+            custom_static_mods = config.static_mods
+
+            if custom_var_mods is not None and not custom_var_mods:
+                update_mod_masses(custom_var_mods.values())
+            if custom_static_mods is not None and not custom_static_mods:
+                update_mod_masses(custom_static_mods.values())
```

Comment:
please move this logic to where it is actually needed! You shouldn't do this here, as it doesn't allow the usage of pp.convert_search with the dictionary itself, which requires loading conversion functions from fundamentals manually as the user, if one wants to use pp.convert_search standalone. Instead, pp.convert_search should simply accept the config.var_mods.